02_03_HW NEW/css/style.css

```diff
@@ -0,0 +1,111 @@
+
+
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400&family=Playfair+Display&display=swap');
+
+
+
+.body {
+    width: 100%;
+    height: 100vh;
+    background-color: #ffffff;
+}
+
+.page {
+    width: 1400px;
+    height: 940px;
+    -webkit-box-shadow: 0 4px 68px rgba(0, 0, 0, 0.1);
+            box-shadow: 0 4px 68px rgba(0, 0, 0, 0.1);
+    background-color: #ffffff;
+    margin: 120px auto;
+    padding-top: 135px;
+    padding-left: 120px;
+}
+
+/* header */
+.header {
+    width: 100%;
+    height: 34px;
+    color: #171717;
+    font-family: 'Playfair Display', serif;
+    font-size: 42px;
+    font-weight: 400;
+    font-style: normal;
```

Comment:
`font-style: normal;` - unnecessary code. Since normal is the default value. Please, check and remove all other rules that are default